src/Chainweb/Version.hs

```diff
@@ -852,8 +852,8 @@ enablePactEvents _ bh = bh >= 2
 
 -- | Bridge support: ETH and event SPV.
 enableSPVBridge :: ChainwebVersion -> BlockHeight -> Bool
-enableSPVBridge Mainnet01 = (>= 2000000) -- TODOTODOTODOTODOTODOTODO
-enableSPVBridge Testnet04 = (>= 1900000) -- TODOTODOTODOTODOTODOTODO
+enableSPVBridge Mainnet01 = (>= 1_275_000) -- 2021-01-14T16:35:58
```

Comment:
Passes my cursory general estimate check.

Reply:
👍  This is what I get:

Mainnet: "2021-01-14 16:35:25 UTC"
Tetsnet: "2021-01-14 17:09:56 UTC"